infra/index.ts

```diff
@@ -7,7 +8,17 @@ import {
   uploadAssetsToBucket,
 } from './resources';
 
-const BUCKET_NAME = 'access-to-finland-demo';
+const stack = pulumi.getStack();
+
+let BUCKET_NAME;
+
+if (stack === 'dev') {
+  BUCKET_NAME = 'access-to-finland-demo';
```

Comment:
Olisiko parempi jos ei-tuotanto resurssit sisältäisivät aina pre/postfixin, eli tämä olisi access-to-finland-dev?

Ja tuon envin voisi ottaa tuosta stack muuttujasta suoraan jolloin ei tarvitse iffitellä, paitsi tieysti tuotantoa varten. :D

Reply:
Jeps, tää oli hyvä kommentti https://virtualfindev.atlassian.net/browse/VFD-81?focusedCommentId=10033

Eli että API-osoitteet, bucketit ja vastaavat koodissa viitatut askat tulisi esim. ympäristömuuttujista jotka sit deplauksen/buildauksen aikana sujautetaan buildille tiedoksi.

Reply:
> Olisiko parempi jos ei-tuotanto resurssit sisältäisivät aina pre/postfixin, eli tämä olisi access-to-finland-dev?
Voisihan se juu näin olla, tagien lisäksi.

Reply:
Voidaan muuttaa näin, että lukee tuosta pulumin outputista ympäristön (stack name). Tosin nykyinen bucketti sitten jää käyttämättömäksi, mikä siis jo deplattu ja se täytyy poistaa, mutta eipä tuo ole ongelma.